interface/patient_file/encounter/find_code_history.php

```diff
@@ -78,20 +78,6 @@ function get_history_codes($pid)
     <title><?php xlt("Find Code History"); ?></title>
```

Comment:
Something bad happened here @bradymiller @stu01509 

![image](https://user-images.githubusercontent.com/46497068/91352888-7fa8f780-e7b8-11ea-8218-b22bd01d7053.png)

Reply:
In my environment running normally, need @bradymiller  to confirm

Reply:
@bradymiller

Reply:
Hi,
I'd be willing to bet that @tywrenn is seeing this since his instance has not imported the ICD10 codes (Administration->Coding->External Data Loads->ICD10->Install).
And that @stu01509 is not seeing an error since has imported the ICD10 codes
If this is the case, then is a bug.

Reply:
And if is a bug, likely pre-existing. I'll test it out later.

Reply:
Confirmed behavior reported by @tywrenn when clicking the following button below Diagnosis Code :